### PR TITLE
CI-940 CI-773

### PR DESCRIPTION
**Pull request by rajeshdigital**
This branch has got accessibility fixes which need to be merged with master.

### CODE REVIEW

File: src/SFA.DAS.Forecasting.Web/Views/Forecasting/Balance.cshtml [hunk 1]
```diff
@@ -122,36 +122,36 @@
                         <th></th>
                     </tr>
                     <tr>
-                        <th style="width: 60px;">Date</th>
+                        <th style="width: 60px;" title="Date">Date</th>
```
Comment (jamesmking) [inline src/SFA.DAS.Forecasting.Web/Views/Forecasting/Balance.cshtml]: This is duplicating the content of the cell. Do we really need it?

File: src/SFA.DAS.Forecasting.Web/Views/Forecasting/Balance.cshtml [hunk 2]
```diff
@@ -122,36 +122,36 @@
...
                         <th class="financial tippy" data-tippy-trigger="click" title="Cost of apprenticeships that have already started and currently being paid for.  This figure also includes any transfers you’ve made. Apprenticeships that have not started or been paid for are not included in this column">
                         Cost of <br />training<br />
-                        <span class="tip-info"><span class="info-title">info</span></span>
+                        <span class="tip-info"><a href="javascript:void();" class="info-title">info</a></span>
                     </th>
                         <th class="financial tippy" data-tippy-trigger="click" title="This column shows the total cost of all completion payments due for the month. The completion payment is shown in the month after the apprentice plans to end their apprenticeship. It is 20% of the total cost of each apprenticeship">
```
Comment (jamesmking) [inline src/SFA.DAS.Forecasting.Web/Views/Forecasting/Balance.cshtml]: Need scope="col" added to these column headings

File: src/SFA.DAS.Forecasting.Web/Views/Estimation/AddApprenticeships.cshtml [hunk 3]
```diff
@@ -74,36 +73,40 @@
                         </div>
 
 
-                        <div class="form-group">
+                        <div class="form-group form-group-error">
```
Comment (jamesmking) [inline src/SFA.DAS.Forecasting.Web/Views/Estimation/AddApprenticeships.cshtml]: Are these hard coded error classes done for a reason?

Comment (rajeshdigital) [reply src/SFA.DAS.Forecasting.Web/Views/Estimation/AddApprenticeships.cshtml]: This was added to implement the error validation in this page.  Someone has checked in the code without implementing the functionality in the backend.  I shall speak to Chris about this.

**Pull request merged**